client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/sidebar.tsx

```diff
@@ -322,44 +322,6 @@ describe( 'Sidebar', () => {
 				expect( domainUpsellTaskFreeFlow ).toBeVisible();
 				expect( domainUpsellTaskBadgeFreeFlow ).toBeVisible();
 			} );
```

Comment:
Could you add a spec to ensure that the task is displayed specifically for the free flow?

Reply:
@Sharpirate that they exist here https://github.com/Automattic/wp-calypso/blob/add/launchpad_remove_choose_domain/client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/task-helper.ts#L174-L180